src/libs/actions/Policy/Tag.ts

```diff
@@ -237,7 +238,11 @@ function importPolicyTags(policyID: string, tags: PolicyTag[]) {
 }
 
 function setWorkspaceTagEnabled(policyID: string, tagsToUpdate: Record<string, {name: string; enabled: boolean}>, tagListIndex: number) {
-    const policyTag = PolicyUtils.getTagLists(allPolicyTags?.[`${ONYXKEYS.COLLECTION.POLICY_TAGS}${policyID}`] ?? {})?.[tagListIndex] ?? {};
+    const policyTag = PolicyUtils.getTagLists(allPolicyTags?.[`${ONYXKEYS.COLLECTION.POLICY_TAGS}${policyID}`] ?? {})?.at(tagListIndex);
```

Comment:
Just to confirm, `tagListIndex` isn't negative in all of the cases in this file?